xdsl/dialects/x86/ops.py

```diff
@@ -1293,7 +1353,8 @@ def custom_print_attributes(self, printer: Printer) -> Set[str]:
 
 class M_M_Operation(Generic[R1InvT], IRDLOperation, X86Instruction, ABC):
     """
-    A base class for x86 operations with a memory reference that's both a source and a destination
+    A base class for x86 operations with a memory reference that's both a source and a
```

Comment:
Unrelated formatting change?

Reply:
Yep I accidentally included it but it makes the lines fit into 88 columns so I think still a good thing to merge.